addons/website_slides/models/slide_slide.py

```diff
@@ -219,7 +219,7 @@ def _compute_category_id(self):
         self.category_id = False  # initialize whatever the state
 
         channel_slides = {}
-        for slide in self:
+        for slide in self.with_context(active_test=True):
```

Comment:
Mmh ... why this fix ? As _compute_category_id is called by the ORM on a record set, it is impacted by this flag ?

Reply:
Yes _compute_category_id is called by the ORM but seems it's fetching inactive slides/category too. Which is not expected behaviour.